tools/matchms/matchms_convert.xml

```diff
@@ -0,0 +1,64 @@
+<tool id="matchms_convert" name="matchms convert" version="@TOOL_VERSION@+galaxy0" python_template_version="3.7">
+    <description>convert mass spectral library</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator"/>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">matchms</requirement>
+    </requirements>
+
+    <command detect_errors="aggressive"><![CDATA[
+        python ${matchms_python_cli}
+    ]]></command>
+
+<configfiles>
+<configfile name="matchms_python_cli">
+from matchms.importing import load_from_msp, load_from_mgf
+from matchms.exporting import save_as_msp, save_as_mgf
+spectra = list(load_from_${spectral_library.ext}("${spectral_library}", ${harmonize_metadata}))
+save_as_${output_format}(spectra, "${converted_library}")
+</configfile>
+</configfiles>
+
+    <inputs>
+        <param label="Spectra file" name="spectral_library" type="data" format="msp,mgf,json" help="Mass spectra file to be filtered." />
+
+        <param label="Spectral library format" name="output_format" type="select"
+            help="Output format of the spectral library.">
+            <option value="msp">msp</option>
+            <option value="mgf">mgf</option>
+        </param>
+
+        <param name="harmonize_metadata" label="Harmonize metadata" type="boolean" truevalue="True" falsevalue="False" checked="true"
+                   help="Harmonize metadata present in the library."/>
+    </inputs>
+
+    <outputs>
+        <data label="${tool.name} from ${spectral_library.ext} to ${output_format} on ${on_string}" name="converted_library">
+            <change_format>
+                <when input="output_format" value="msp" format="msp" />
+                <when input="output_format" value="mgf" format="mgf" />
+                <when input="output_format" value="json" format="json" />
```

Comment:
it con not be json, isn't it?

Reply:
Yeah, the `output_format` is not allowed to be `json` - either remove it here and allow it as an output format.

Reply:
Yes, it can be - I re-added the json part back in. I had it removed for some time since it didn't handle some files, but now it seems okay - I disabled the metadata harmonization and now it seems to work.